prefs.js

```diff
@@ -31,6 +31,26 @@ const TimezoneExtensionPrefsWidget = new GObject.Class({
                                  use_markup: true, margin_bottom: 6,
                                  hexpand: true, halign: Gtk.Align.START}));
 
+        let box = new Gtk.Box({orientation: Gtk.Orientation.HORIZONTAL, spacing: 5});
+
+        box.add(new Gtk.Label({ label: '<b>Highlight working hours</b>',
+                                 use_markup: true, margin_bottom: 6,
+                                 hexpand: false, halign: Gtk.Align.START }));
+
+        this._hour1 = Gtk.SpinButton.new_with_range (0, 23, 1);
+        this._hour2 = Gtk.SpinButton.new_with_range (0, 23, 1);
+
```

Comment:
I think this might cause problems with US users, which use the am-pm approach. We can proceed with this and improve it later, in another issue if you want.